src/lib/swapper/swappers/CowSwapper/utils/helpers/helpers.ts

```diff
@@ -227,6 +201,17 @@ export const getValuesFromQuoteResponse = ({
   )
 
   const rate = bnOrZero(buyAmountAfterFeesCryptoPrecision).div(sellAmountCryptoPrecision).toString()
```

Comment:
No need to use the native price endpoint (a la https://github.com/shapeshift/web/issues/5099), we have enough information already.

Reply:
hell yes